source/PokerSetup.py

```diff
@@ -0,0 +1,344 @@
+                #   Call Check Function.
+                PokerPlayer.Check()
+
+            #   Player Calls!
+            elif Move == "Call" or Move == "CALL" or Move == "call":
+                #   Call Call Function.
+                PokerPlayer.Call()
+
+            #   Player Raises!
+            elif Move == "Raise" or Move == "RAISE" or Move == "raise":
+                #   Call Raise Function.
+                PokerPlayer.Raise()
+
+            #   Player is All In!
+            if Move == "All In" or Move == "ALLIN" or Move == "ALL IN" or Move == "All in" or Move == "all In" or Move == "all in" or Move == "allin":
+                #   Call Check Function.
+                PokerPlayer.Check()
+
+            #   Player Folds!
+            elif Move == "Fold" or Move == "fold":
+                #   Call Fold Function.
+                PokerPlayer.Fold()
+
+            #   If an invalid move was entered.
+            else:
+                print('''You did not corretly enter a valid move.
+Automatic Fold!''')
+                PokerPlayer.Fold()
+
+    def Check(self):
+        #   Checks and passes.
+        pass
+
+    def Call(self):
+        #   Call and matches bidding amount.
+        pass
+
+    def Raises(self):
+        #   Raises and increases bidding amount.
+        #
+        #   I think we are doing it where the raise has to be at least double the preious raise.
+        #
+        #   If raised, then another loop must be factored in for betting that round until it arrives at the last person to raise it.
+        #
+        #   How I have played Texas Hold Em is that the person who raised it in a round, cannot raise it again in the same round
+        #   This will work because then we don't have to factor in the potential scenarior where it loops around more times than there are players playing.
+        #   So loop through the bets, if someone Raises, begin a new loop until it ends at the person who is before the person who raised it.
+        pass
+
+    def allIn(self):
+        #   Raises bidding amount with all the money the player has left.
+        #
+        #   Technically, when someone goes all in, those who are wealtheier can still outbid him.
+        #   So the player who goes all in will have to leave PlayersPlayingRoundList.
+        #   But if he wins, they get placed back in.
+        #   They only win all the player's money that matches them.
+        #   If there is more money in the pot, then all the extra money goes back to original owners.
+        #   If they lose, they drop out of the game entirely.
+        pass
+
+    def Fold(self):
+        #   Fold and leaves PlayersPlayingRoundList.
+        pass
+
+    def addCardToRiver(self):
+        #   Adds a card to the River and Displays River!
+        #   Made a river list.
+        pass
+
+    def playFirstRound(self):
+        #   First Round of a Game.
+        PokerGameSetup.Blinds(self)
+        PokerGameRounds.startRound(self)
```

Comment:
Since this is the PokerGameRounds class, the proper way to call this would be "self.startRound()"